### DISCUSSION TITLE
Older iOS app support?

### DISCUSSION THREAD

**Discussion by darkner258** [music-assistant/support#3114] [Q&A] [answered]
### What version of Music Assistant has the issue?

2.3.2

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

Hey guys, first of all, love the addon! Great to see it can even play on my Belkin Soundform Connect, which none of the others can!

I have an old iPad I use as a dashboard and the HA app won't let me update past app version 2023.12.1. 
On this version, MA does not work, only shows a blank screen. Is there a way to get around that?

### How to reproduce

Old iOS device

### Music Providers

Apple Music

### Player Providers

Airplay

### What version of Home Assistant Core are your running

2024.10.4

### What type of installation are you running?

Home Assistant Core

### On what type of hardware are you running?

Linux

**Comment by OzGav** [comment]
Moved to discussion as this is not a MA problem.

**Reply by OzGav** [reply]
Edit: Or are you saying the browser is incompatible?

**Reply by darkner258** [reply]
> Edit: Or are you saying the browser is incompatible?

Yup, seems like it. The rest of the app works fine, just the MA frontend won't load. Running it somewhere else doesn't really make sense in my case, as the goal is to keep everything in one app (HA app). 
Sidenote, same happens in Safari browser

**Reply by OzGav** [reply, marked as answer]
Yes because on IOS all browsers use the same underlying engine. Unfortunately, I am unaware of any way to make this work. Like.y this is why the app won’t update as well.

**Reply by darkner258** [reply]
That's a big shame, but thanks for the answer anyway!
